vita3k/gui/src/settings_dialog.cpp

```diff
@@ -455,7 +455,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
 #endif
         ImGui::SetCursorPosX((ImGui::GetWindowWidth() / 2.f) - (ImGui::CalcTextSize("Resolution Upscaling").x / 2.f));
-        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Resolution Upscaling");
+        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Internal Resolution Upscaling");
```

Comment:
wrong nee dchange for both, look upp line

Reply:
The upscaling is indeed by altering the internal resolution, change is correct

Reply:
i no speek this, i speek need change two line..., if you change string, need change also calctextsize line